tests/kwio_test.sh

```diff
@@ -104,10 +106,19 @@ function alert_completition_validate_config_file_options_Test
 function alert_completion_visual_alert_Test
 {
   local expected="TESTING COMMAND"
-  configurations["visual_alert_command"]="echo \$COMMAND"
+  configurations["visual_alert_command"]="/bin/echo \$COMMAND"
   ret="$(alert_completion "$expected" "--alert=v")"
   assertEquals "Variable $v should exist." "$ret" "$expected"
   true
 }
 
+function alert_completion_sound_alert_Test
+{
+  local expected="TESTING COMMAND"
+  configurations["sound_alert_command"]="/bin/echo \$COMMAND"
+  ret="$(alert_completion "$expected" "--alert=s")"
```

Comment:
Maybe `output` instead of `ret`. Also, declare it as a `local` variable and all other variables that belong to this specific test.